tracer/test/snapshots/Iast.EmailHtmlInjection.AspNetCore2.IastEnabled.verified.txt

```diff
@@ -0,0 +1,96 @@
+﻿[
```

Comment:
In these tests, evidence redaction is not enabled

Reply:
Just to check I've understood, to avoid adding an excessive number of integration tests, we just test with IAST enabled and redaction disabled?

We disable the redaction because this behaviour is already checked with the redaction test suite?

Reply:
Yes, the evidence redaction is tested in the evidence redaction unit tests. On the other side, testing without evidence redaction is useful to check that the information regarding the vulnerability is processed. In order to avoid too many integration tests, evidence redaction is not tested here.